variables.tf

```diff
@@ -0,0 +1,134 @@
+variable "additional_iam_policy_arns" {
+  default     = []
+  description = "List of ARN policies for addional builder permissions"
+  type        = list(string)
+}
+
+variable "description" {
+  default     = null
+  description = "description of component"
+  type        = string
+}
+
+variable "instance_types" {
+  default     = ["t3.medium"]
+  description = "Instance types to create images from. It's unclear why this is a list. Possibly because different types can result in different images (like ARM instances)"
```

Comment:
It's so that you have a greater shot at getting an instance in spot. A lot of people use spot for this sort of thing because it's not a big deal to just retry the build if you get booted. Might be a good future feature for us.

Reply:
ah. Can you set up EC2 Image Builder to use spot? I didn't see anything that would allow you to specify that.

Reply:
not explicitly but I wouldn't be surprised if it uses it under the hood..